core/src/components/input/test/basic/input.e2e.ts

```diff
@@ -180,9 +180,7 @@ configs({ modes: ['ionic-md'] }).forEach(({ title, screenshot, config }) => {
       await input.evaluate((el: HTMLIonInputElement) => el.setFocus());
       await page.waitForChanges();
 
-      const clearButton = input.locator('.input-clear-icon');
-      clearButton.evaluate((el: HTMLElement) => el.classList.add('ion-focused'));
-      await page.waitForChanges();
+      await pageUtils.pressKeys('Tab');
 
       const container = page.locator('#container');
       await expect(container).toHaveScreenshot(screenshot(`input-clear-button-focused`));
```

Comment:
How are these screenshots not updated by this PR? https://github.com/ionic-team/ionic-framework/blob/ROU-4848/core/src/components/input/test/basic/input.e2e.ts-snapshots/input-clear-button-focused-ionic-md-ltr-light-Mobile-Chrome-linux.png

Reply:
The screenshots were not updated because no visual changes happened. The focus ring is still the same shape, size, color, etc.